storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -70,6 +70,80 @@ export const Default: StoryObj = {
   },
 }
 
+export const WithTopTasks: StoryObj = {
+  args: {
+    children: [
+      <Grid key={1} paddingTop="large">
+        <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+          <Breadcrumb>
+            <Breadcrumb.Link href="#">Home</Breadcrumb.Link>
+          </Breadcrumb>
+        </Grid.Cell>
+      </Grid>,
+      <Grid as="main" id="inhoud" key={2} paddingBottom="x-large">
+        <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }}>
+          <Heading className="ams-mb-m" level={1}>
+            Leefomgeving
+          </Heading>
+        </Grid.Cell>
+        <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
+          <Card>
+            <Card.Heading level={2}>
```

Comment:
The heading hierarchy is off on this page, can you check it in HeadingsMap (or something similar)?